client/core/core.go

```diff
@@ -3582,15 +3582,47 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
 					notifyErr(SubjectMatchStatusError, "Match %s for order %s is in state %s, but has no maker swap contract.", dbMatch.Side, tracker.token(), dbMatch.Status)
 					continue
 				}
+				// Obtaining AuditInfo will fail if it's unmined AND gone from
+				// mempool, or the wallet is otherwise not ready. Note that this
+				// does not actually audit the contract's value, recipient,
+				// expiration, or secret hash (if maker), as that was already
+				// done when it was initially stored as CounterScript.
 				auditInfo, err := wallets.toWallet.AuditContract(counterSwap, counterContract)
 				if err != nil {
-					c.log.Debugf("AuditContract error for match %v status %v, refunded = %v, revoked = %v: %v",
-						match.id, match.MetaData.Status, len(match.MetaData.Proof.RefundCoin) > 0, match.MetaData.Proof.IsRevoked(), err)
+					contractStr := coinIDString(wallets.toAsset.ID, counterSwap)
+					c.log.Warnf("Starting search for counterparty contract %v (%s)", contractStr, unbip(wallets.toAsset.ID))
```

Comment:
I tested this for both btc and dcr contracts (mined and unmined) by restarting the wallets and nodes, also by forcing the goroutine search by changing the condition to `err == nil`.  I'm not sure if restarting the wallets and nodes forced it though because I added these two log statements after that but before the hacked code test.

Reply:
While the search is ongoing, trace logs say:

> 2021-02-04 15:02:20.396 [TRC] CORE: Match 9deab745f96c0cd7398ef1a09fab4e68c6627df2ccec402a352a8664e42fa714 not swappable: swapErr = audit in progress, please wait, revoked = false, metered = false
> 2021-02-04 15:02:20.396 [TRC] CORE: Match 9deab745f96c0cd7398ef1a09fab4e68c6627df2ccec402a352a8664e42fa714 not redeemable: swapErr = audit in progress, please wait, RefundCoin = , metered = false

tested that with 

```diff
diff --git a/client/core/core.go b/client/core/core.go
index a3d2cf61..7b149d07 100644
--- a/client/core/core.go
+++ b/client/core/core.go
@@ -3588,7 +3588,7 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
                                // expiration, or secret hash (if maker), as that was already
                                // done when it was initially stored as CounterScript.
                                auditInfo, err := wallets.toWallet.AuditContract(counterSwap, counterContract)
-                               if err != nil {
+                               if err == nil {
                                        contractStr := coinIDString(wallets.toAsset.ID, counterSwap)
                                        c.log.Warnf("Starting search for counterparty contract %v (%s)", contractStr, unbip(wallets.toAsset.ID))
                                        // Start the audit retry waiter. Set swapErr to block tick
@@ -3597,6 +3597,7 @@ func (c *Core) resumeTrades(dc *dexConnection, trackers []*trackedTrade) assetMa
                                        match.swapErr = fmt.Errorf("audit in progress, please wait") // don't frighten the users
                                        go func(tracker *trackedTrade, match *matchTracker) {
                                                auditInfo, err := tracker.searchAuditInfo(match, counterSwap, counterContract)
+                                               time.Sleep(time.Minute)
                                                tracker.mtx.Lock()
                                                defer tracker.mtx.Unlock()
                                                if err != nil {
```